### ISSUE TITLE
p-multiselect (accessibility issue): checkbox under .ui-helper-hidden-accessible  cannot be defined aria-label.

### ISSUE THREAD

**Issue by qiong253112** [primefaces/primeng#8867]
**Current behavior**
 Checkbox under .ui-helper-hidden-accessible  cannot be defined aria-label. If we use the accessibility insight for web tool to test then will get the following issue:
![image](https://user-images.githubusercontent.com/29710072/82648872-f333e980-9c4a-11ea-8a82-fa3d7a600ae6.png)

**Expected behavior**
 Checkbox under .ui-helper-hidden-accessible should be defined aria-label
Effects programmatic accessibility of the control.

**Reference Links**
[Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by yigitfindikli** [comment]
Unable to replicate with PrimeNG 11.0.0-rc.1, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.
